apm-agent-plugins/apm-profiling-plugin/src/test/java/co/elastic/apm/agent/profiler/CallTreeTest.java

```diff
@@ -59,7 +59,6 @@
 import static org.mockito.Mockito.doReturn;
 
 @DisabledOnOs(OS.WINDOWS)
-@DisabledOnAppleSilicon
```

Comment:
Ooops!

Reply:
@felixbarny I am restoring on a separate PR unless you tell me this is intentional

Reply:
Whoops, definitely not intentional.